src/esp/assets/CMakeLists.txt

```
@@ -79,8 +79,11 @@ target_link_libraries(
          MagnumPlugins::StbImageConverter
          MagnumPlugins::TinyGltfImporter
   PRIVATE geo io
+         vhacd
 )
 
+target_include_directories(vhacd PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../../deps/v-hacd/src/VHACD_Lib/public)
```

Modifying targets from outside is allowed only in CMake 3.13 and newer. I don't think Habitat requires that version yet, and I would suggest not arbitrarily increasing the required CMake version unless absolutely needed -- it adds unnecessary friction for users on LTS distros :) Ubuntu 18.04 LTS has 3.10, so I'd say staying at that version is reasonable. (Just FYI Magnum has stricter requirements, so there the minimal version is still 3.4.)

To make this work on older CMakes, I'd instead do

```suggestion
target_include_directories(assets PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../../deps/v-hacd/src/VHACD_Lib/public)
```

... and ideally fork / submit a patch to vhacd so it does a `target_include_directories()` on its own :)